package.json

```diff
@@ -148,11 +148,13 @@
     "cheerio": "^1.0.0-rc.10",
     "componentsjs-generator": "^2.6.1",
     "eslint": "^8.8.0",
-    "eslint-config-es": "4.1.0",
+    "eslint-config-es": "^4.1.0",
```

Comment:
This one was fixed on purpose though, as that library tends to sneak breaking changes into minor/patch releases causing our linter to suddenly fail.